runtime/src/objc/cpp/ObjCExportNumbers.mm

```diff
@@ -270,6 +270,10 @@ @implementation KotlinBoolean {
   BOOL value_;
 }
 
+- (void)getValue:(void *)value {
```

Comment:
These classes are actually generated with "script" above: https://github.com/JetBrains/kotlin-native/blob/e8ef64764cf8ff30c911d3b37d2d3b433ea9c47c/runtime/src/objc/cpp/ObjCExportNumbers.mm#L103
So it would be great to fix the script too.

Reply:
Done